include/sensors/core/tasks/pressure_driver.hpp

```diff
@@ -323,6 +330,7 @@ class MMR920 {
             _registers.pressure_result.reading, sensor_version);
 
         if (max_pressure_sync) {
+            sensor_buffer_log(pressure);
```

Comment:
This doesn't go inside this if block, it should be the `if(echo_this_time) ` block otherwise we are double logging the data
and we can remove the ifdef from the` if(echo_this_time)` block